bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -131,23 +132,23 @@
 		<label>Temperature</label>
 		<description>Current temperature</description>
 		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"></state>
+		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 
 	<channel-type id="humidity">
-		<item-type>Number</item-type>
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
Now that you are at it, you could consider removing this custom channel type and instead reference `atmospheric-humidity` from the respective channels:

https://github.com/openhab/openhab-core/blob/c9ee9920478c783ea4f40dacab79ea41569ef505/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java#L339-L346

If you don't like the label and/or description, you can of course override it, but it should then also be overridden in the update instructions.

The system default channel type is formatted without any decimals, but I doubt the measurement is actually sufficiently accurate anyway?

Reply:
It would have to override the label, description and state description, and make the update instruction more complex and potentially confusing. So if it is Ok with you I would rather leave this one as it is.

Reply:
Why would it need to override state description (which is not possible)? Will just leave the comment that the only thing needed for the update instructions would be copying the overridden label/description from the thing type definition, for example:
```xml
			<update-channel id="humidity">
				<type>tado:humidity</type>
                                <label>Humidity</label>
                                <description>Current humidity in %</description>
			</update-channel>
```

You would get semantic tags (`Measurement`, `Humidity`) for free. But as said, this was only for consideration.

Reply:
Ok. You convinced me. Please note that in the thing.xml the channel type is `system.atmospheric-humidity` (with a dot) whereas in the upgrade instructions I had to enter it as `system:atmospheric-humidity` (with a colon) -- this seems to be inconsistent but apparently correct as the xsd schema won't accept a channel type with a dot.